### PR TITLE
Merging flash shields into riot shield type, few balance tweaks 

### PR DESCRIPTION
**Pull request by Sparkezel**
## About The Pull Request

Out of curiosity I wanted to check whenever you could put in hypnoflashes into strobe shields or not and I got very dissapointed that no, you couldn't. Afterwards I tried to change that, decided that you should also make it possible to make flash bucklers and.... well... here we are.

Now all shields under riot shield subtype (which is riot shield, telescopic, buckler) can be equipped with a flashbulb mount and you can put a flash or a hypnoflash inside! You can remove the flash using wirecutters and the flashbulb mount using a crowbar.

I also gave the fake roman shield same block ability as a real one, but I also made it more flimsy and made each shield have its own, more sensible fixing material than goddamn titanium sheets.

## Why It's Good For The Game
Strobe shield is no longer a snowflaked object, you can convert most kinds of riot shields into all kinds of strobe shields.
You couldnt install hypnoflashes into strobeshields due to how flash subtypes were made, now you can.
You no longer need to use titanium sheets to fix your wooden buckler.

## What needs to be done yet

- [x] Overlays for shields
- [x] Potential fix of flashing animation for inhands
- [x] Inhand sprites
## Changelog
:cl:

expansion: You can now turn buckler and telescopic shields into strobe shields. Telescopic shields toggle now on alt. You can bash people with strobeshields using rmb/disarm.
del: Deleted strobe shield recipe
balance: Each shield has its own fixing material, rglass for telescopic and riot, wood for buckler, iron for roman
balance: Rebalanced fake roman shield to have same block value as real one but made it more flimsy
imageadd: added overlays for flash mount and flash on shields, added inhands for stroboscopic shields
code: Changed hypnoflash to be of flash/handheld type
code: Prettied up shieldcode
/:cl:

### CODE REVIEW

File: code/game/objects/items/shields.dm [hunk 1]
```diff
@@ -27,32 +90,107 @@
...
+			playsound(src, 'sound/items/deconstruct.ogg', 50, TRUE)
+			if(embedded_flash)
+				embedded_flash.forceMove(get_turf(src))
+			embedded_flash = flash
+			flash.forceMove(src)
+			update_appearance()
+		return
+
+	if (W.tool_behaviour == TOOL_WIRECUTTER)
+		if(embedded_flash)
```
Comment (EdgeLordExe) [inline code/game/objects/items/shields.dm:169]: you can make these into a single if statement

**Review by EdgeLordExe — COMMENTED** [review]
besides this LGTM

File: code/game/objects/items/shields.dm (hunk 1 above)
Comment (Sparkezel) [reply code/game/objects/items/shields.dm]: done

**Comment by Sparkezel** [comment]
image
it looks like dm doesnt like that

**Comment by Sparkezel** [comment]
accidental early return

**Pull request merged**